src/engine/Test_dataflow_tainting.ml

```diff
@@ -34,7 +34,7 @@ let test_tainting lang file options config def =
           let tok1, tok2 = (fst (Taint.pm_of_trace src.call_trace)).range_loc in
           let r = Range.range_of_token_locations tok1 tok2 in
           Range.content_at_range file r
-      | Taint.Arg (s, i) -> spf "arg %s %d" s i
+      | Taint.Arg arg -> Taint._show_arg arg
```

Comment:
Out of curiosity why the underscore?

Reply:
It's just a "debugging version" of `show_arg`, in this case it could probably replace `show_arg` because it contains the same information and it's prettier. But I tend to leave the derived `show_arg` as they are.